kedro_mlflow/framework/cli/cli.py

```diff
@@ -1,16 +1,12 @@
-import os
 import subprocess
 from pathlib import Path
 
 import click
-from kedro import __file__ as KEDRO_PATH
+from kedro.framework.context import get_static_project_data, load_context
```

Comment:
I think this breaks retrocompatibilty with kedro<=0.16.4. It should definitely be a separated PR in case we need to rollback. But it's a great idea!

Reply:
That will definitely break kedro <=0.16.4 compatibility as i just saw that `get_static_project_data` is introduced in kedro 0.16.5. I'll keep the `_get_project_globals` utils